about_binaries.exs

```diff
@@ -31,4 +31,65 @@ defmodule AboutBinaries do
     contatenated = "Hello " <> "world"
     assert contatenated == __?
   end
+
+  @tag :focus
+  think "Numbers can be represented as bits" do
+    bit_representation = fn (integer) -> Integer.to_string(integer, 2) end
```

Comment:
Better to DRY this up? I lean toward duplication here so that each koan stands alone and no scrolling is required to understand.